web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -781,4 +831,8 @@ class AppConfig extends Component {
   }
 }
 
-export default withRouter(AppConfig);
+// TODO: fix this type
+// eslint-disable-next-line
+const AppConfigWithRouter: any = withRouter(AppConfig);
+
+export default AppConfigWithRouter;
```

Comment:
This is a mess.  I need to dig into this to figure out how to type components decorated with react router

Reply:
don't know if this is helpful but from researching, it seems like you can use RouteComponentProps https://www.reactandtypescript.dev/examples/react-router#withrouter